src/index.test.js

```diff
@@ -528,15 +541,15 @@ describe("Lumigo plugin (python)", () => {
 					serverless.service.custom.lumigo.pythonLayerVersion = 87;
 				});
 
-				test("layer version 87 are added during after:package:initialize", async () => {
-					await lumigo.afterPackageInitialize();
+				test("layer version 87 are added during after:package:createDeploymentArtifacts", async () => {
+					await lumigo.afterCreateDeploymentArtifacts();
 
 					assertPythonFunctionsHaveLayers(87);
 				});
 
-				test("layers are added during after:deploy:function:initialize", async () => {
+				test("layers are added during after:package:createDeploymentArtifacts", async () => {
 					options.function = "hello";
-					await lumigo.afterDeployFunctionInitialize();
+					await lumigo.afterCreateDeploymentArtifacts();
 
 					const functions = serverless.service.functions;
 					expect(functions.hello.handler).toBe(
```

Comment:
Can you also add an integration-test? Check the file `scripts/checks.sh` and the directory `integration-test`.

Reply:
Nice !!!